File: src/update_xml.pl
#!/usr/bin/perl

$root_path=$ENV{ROOT_PATH};
$file=$ARGV[0];

$full_text="";
open(O, "<$file");
while($r=<O>) {
$full_text.=$r;
}
close(O);

open(O, ">$file");
$full_text=~s/__DBNAME__/$ENV{DB_NAME}/g;
print O $full_text;
close(O);

File: src/update_xml.php
#!/usr/bin/php
<?
$root_path=$_ENV['ROOT_PATH'];
$file=$argv[1];

$full_text="";
$full_text=file_get_contents($file);

$tr=array();
$tr["__DBNAME__"]=$_ENV['DB_NAME'];

$full_text=strtr($full_text, $tr);

$f=fopen("$file", "w");
fwrite($f, $full_text);
fclose($f);
